utils/histogram.hh

```diff
@@ -258,7 +258,17 @@ public:
         rate_moving_average res;
         if (_count > 0) {
             double elapsed = std::chrono::duration_cast<std::chrono::seconds>(latency_counter::now() - start_time).count();
-            res.mean_rate = (_count / elapsed);
+            double mean_rate = (_count / elapsed);            
+            [[unlikely]]
+            if ((elapsed < 1.0) && std::isinf(mean_rate)) {
```

Comment:
This is wrong, it says the if statement is unlikely (both branches).

Reply:
Thanks, I will move it to the appropriate place.